test/src/817-gateway_abort/main

```diff
@@ -0,0 +1,26 @@
+cvmfs_test_name="Lease cleanup when the publishing process aborts on the gateway"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="quick"
+
+
+cvmfs_run_test() {
+    set_up_repository_gateway || return 1
```

Comment:
`set_up_repository_gateway` creates `test.repo.org`. this is non-obvious. maybe we should either use `$CVMFS_TEST_REPO` or define a variable `CVMFS_GATEWAY_TEST_REPO=test.repo.org` which should always be used inside gateway integration tests.

and maybe change add the name of the repo to https://github.com/cvmfs/cvmfs/blob/a4512f998399660c1fda99297ceb979b0a5d2822/test/test_functions#L2533

Reply:
Agreed. I'd prefer to address this in a different PR/issue: https://github.com/cvmfs/cvmfs/issues/3340